MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml

```diff
@@ -662,7 +728,7 @@
 
     <!-- This string is for the tooltip that appears upon hovering the 'User Agent' button in the
```

Comment:
The comment here needs to be updated.
Also, since the string changes drastically in meaning, the string ID will need to be updated so that localizers can see it has changed.

The strings that have been modified below all need to be fixed in the same way (so, updating localization comment as well as string ID).

Reply:
All these are new strings in 1.4 and haven't been translated yet (at least the translations haven't been comited to the FxR repo yet). Do we still need to add new strings even if they have only changed before being translated?

Reply:
Comment updated.

Reply:
Hmm  I see it landed and localized on our side. See French here: https://pontoon.mozilla.org/fr/android-l10n/MozillaReality/FirefoxReality/app/src/main/res/values/strings.xml/?search=user_agent_tooltip&string=200838